examples/efs/filesystem_encrypted.yaml

```
@@ -0,0 +1,26 @@
+apiVersion: efs.aws.upbound.io/v1beta1
+kind: FileSystem
+metadata:
+  name: example
+  labels:
+    testing.upbound.io/example-name: efs
+spec:
+  forProvider:
+    region: us-west-1
+    creationToken: my-product
+    tags:
+      Name: MyProduct
+    kmsKeyIdRef:
+      name: efs-key
+---
+apiVersion: kms.aws.upbound.io/v1beta1
+kind: Key
+metadata:
```

```suggestion
metadata:
  annotations:
    meta.upbound.io/example-id: efs/v1beta1/filesystem
```
Could you please add the example id?